src/IOHandler.java

```diff
@@ -0,0 +1,107 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class IOHandler {
```

Comment:
it might have made sense to even separate file i/o from terminal i/o, since now this class won't be usable in a GUI version (or at least the system.out.println commands would be useless)